src/audit/cache_poisoning.rs

```diff
@@ -110,13 +111,27 @@ static KNOWN_CACHE_AWARE_ACTIONS: LazyLock<Vec<CacheAwareAction>> = LazyLock::ne
     ]
 });
 
+/// A list of well-know publisher actions
+/// In the future we can retrieve this list from the static API
+static KNOWN_PUBLISHER_ACTIONS: LazyLock<Vec<Uses>> = LazyLock::new(|| {
+    vec![
+        Uses::from_step("pypa/gh-action-pypi-publish").unwrap(),
+        Uses::from_step("softprops/action-gh-release").unwrap(),
```

Comment:
Let's add the Ruby one too:

```suggestion
        Uses::from_step("softprops/action-gh-release").unwrap(),
        Uses::from_step("rubygems/release-gem").unwrap(),
```